Sources/Orbit/Support/Components/TimelineStep.swift

```diff
@@ -47,61 +53,97 @@ public struct TimelineStep<Header: View, Content: View>: View {
+                    .overlay(icon)
+            }
     }
 
     @ViewBuilder var icon: some View {
         switch style {
-            case .default:
+            case .inactive:
                 Circle()
-                    .strokeBorder(Color.cloudNormalHover, lineWidth: 2)
+                    .strokeBorder(style.color, lineWidth: 2)
+                    .background(Circle().fill(Color.whiteNormal))
                     .frame(width: .small * sizeCategory.ratio, height: .small * sizeCategory.ratio)
-                    .background(
-                        Circle()
-                            .fill(Color.whiteNormal)
-                    )
-            case .status:
-                Icon(style.iconSymbol, size: .large, color: style.color)
-                    .background(
-                        Circle()
-                            .fill(Color.whiteNormal)
-                            .padding(.xxSmall + 1)
-                    )
+            case .currentNormal:
+                ZStack {
+                    Circle()
+                        .frame(
+                            width: (animationLoopTrigger ? .medium : .xMedium) * sizeCategory.ratio,
+                            height: (animationLoopTrigger ? .medium : .xMedium) * sizeCategory.ratio
+                        )
+                        .animation(Animation.easeInOut.repeatForever().speed(animationSpeed))
+                        .foregroundColor(style.color.opacity(0.1))
+
+                    Circle()
+                        .strokeBorder(style.color, lineWidth: 2)
+                        .background(Circle().fill(Color.whiteNormal))
+                        .frame(width: .small * sizeCategory.ratio, height: .small * sizeCategory.ratio)
+
+                    Circle()
+                        .frame(width: .xxSmall * sizeCategory.ratio, height: .xxSmall * sizeCategory.ratio)
+                        .scaleEffect(animationLoopTrigger ? .init(width: 0.5, height: 0.5) : .init(width: 1, height: 1))
+                        .foregroundColor(animationLoopTrigger ? Color.clear : style.color)
+                        .animation(Animation.easeInOut.repeatForever().speed(animationSpeed))
+
+                }
+            case .currentWarning, .currentCritical, .pastSuccess, .pastWarning, .pastCritical:
+                Icon(style.iconSymbol, size: .small, color: style.color)
+                    .background(Circle().fill(Color.whiteNormal).padding(2))
         }
     }
 }
 
 // MARK: - Inits
 
-public extension TimelineStep where Header == TimelineStepBadgeText {
+public extension TimelineStep where Header == TimelineStepHeadingText {
 
     /// Creates Orbit TimelineStep component with custom details.
     init(
         _ label: String,
         sublabel: String = "",
-        style: TimelineStepStyle = .default,
-        @ViewBuilder content: () -> Content
+        style: TimelineStepStyle = .inactive,
+        @ViewBuilder content: () -> Content,
```

Comment:
I thought we could only rename the existing `content` -> `footer`, not requiring to support both at the same time. But maybe we do need both? 🤔

Similar to components like ListChoice, call site can leave some fields empty and use custom content or it can use both. Can similar api work here as well?

Reply:
Left only `footer`